docs/source/model_design/vivarium_overview/data_sources/index.rst

```diff
@@ -134,29 +134,99 @@ For reference, `the decomposition rules for GBD 2021 can be found here <https://
 
 Additionally, you may be required to specify a :code:`version_id`, :code:`release_id`, and/or :code:`status` when pulling GBD results from certain GBD rounds. The HUB space for a given GBD round is a good resource on where to obtain this information, but do not hesitate to open a helpdesk ticket to inquire or confirm whether you are using appropriate versioning IDs for you GBD shared functions call.
 
+.. todo::
+
+   Discuss release_id as preferred alternative to gbd_round_id + decomp_step.
+
 Pulling GBD Data using *Vivarium Inputs*
 ----------------------------------------
 
+There are two main code bases within the Vivarium software framework that are especially useful for interacting with GBD data, including `gbd_mapping <https://vivarium.readthedocs.io/projects/gbd-mapping/en/latest/index.html>`_ and `vivarium_inputs.interface <https://vivarium.readthedocs.io/projects/vivarium-inputs/en/latest/index.html>`_.
+
+Both of these code bases translate ID numbers used in GBD to human-readable text by default.
```

Comment:
I'll remove "by default" - I initially included a modifier here because there are some methods you can use that do not reformat so that you can view the underlying IDs, but that's not it's typical behavior.